relay-server/src/services/store.rs

```diff
@@ -1350,6 +1351,8 @@ struct SpanKafkaMessage<'a> {
     start_timestamp: f64,
     #[serde(rename(deserialize = "timestamp"), skip_serializing)]
     end_timestamp: f64,
+    #[serde(default, rename(deserialize = "tags"), skip_serializing)]
+    annotated_tags: Vec<(String, String)>,
```

Comment:
> `rename(deserialize = "tags")`

Does this field conflict with `tags` below? Which of the two is produced to Kafka?

Reply:
This is what I want to deserialize as `tags` indeed, and that's why I added the `skip_deserializing` above.

The annotated protocol represents the `Tags` struct as pairs of strings and that's why I need to deserialize like this and then repack it as a map. It's a little bit of mental gymnastic to set the serde annotations correctly but I think they're right.